src/main/java/core/basesyntax/BinaryString.java

```diff
@@ -7,6 +7,18 @@
      * а возвращает String с представлением этого числа в двоичном виде.
      */
     public String toBinaryString(int value) {
-        return null;
+        if (value == 0) {
+            return "0";
+        }
+        StringBuilder binaryString = new StringBuilder("");
+        while (value > 0) {
+            if ((value % 2) == 0) {
+                binaryString.append(0);
+            } else {
+                binaryString.append(1);
+            }
```

Comment:
binaryString.append(value % 2);
```suggestion
            binaryString.append(value % 2);
```

Reply:
Done